spec/integration/thousand_spec.rb

```diff
@@ -0,0 +1,88 @@
+require 'open3'
+
+module RSpec::Core
+  RSpec.describe "Rspec", :slow do
+    def measure(cmd, *args)
+      Dir.mktmpdir do |dir|
+        yield dir
+        start = Time.now
+        path = File.absolute_path(Bundler.which(cmd))
```

Comment:
You'll need to `require 'bundler'` (again our of the main process) if you want to do this, as we isolate it to improve speed by default.